sklearn/manifold/t_sne.py

```diff
@@ -672,6 +676,12 @@ def __init__(self, n_components=2, perplexity=30.0,
         self.method = method
         self.angle = angle
 
+        # TODO sklearn 0.20: remove this warning
+        if learning_rate == 200.0:
```

Comment:
People will probably find strange ways to turn this warning off. Maybe that means setting `learning_rate=199.99` which is certainly worse.